CHANGELOG.md

```diff
@@ -0,0 +1,130 @@
+# TiKV Change Log
+All notable changes to this project are documented in this file.
+See also [TiDB change log][tidb_change_log] and [PD change log][pd_change_log].
+
+[tidb_change_log]: https://github.com/pingcap/tidb/blob/master/CHANGELOG.md
+[pd_change_log]: https://github.com/pingcap/pd/blob/master/CHANGELOG.md
+
+## [2.0.0] - 2018-04-27
+### New Features
+* `tikv-ctl` now can recover MVCC data corruption.
```

Comment:
I think the period in all the bulleted list items is unnecessary, to keep it consistent with other repos.